src/test/app/common/calculate-month-increment/calculate-month-increment.ts

```diff
@@ -84,11 +84,19 @@ describe('calculateMonthIncrement', () => {
     expect(calculateMonthDate.toString()).to.equal(MomentFactory.parse('2019-01-31').toString())
   })
 
-  it('should return null when given a null start date', () => {
-    expect(calculateMonthIncrement(null)).to.be.equal(null)
+  it('should return error when given a null start date', () => {
+    try {
+      calculateMonthIncrement(null)
+    } catch (e) {
```

Comment:
Can we please use `.to.throw(Error, 'Start Date is invalid')`, please refer to other test and try to be consistent